src/client.ts

```diff
@@ -289,21 +289,15 @@ export default class Client implements ClientInterface {
       ),
       vscode.commands.registerCommand(
         Command.DebugTest,
-        this.debugTest.bind(this)
+        this.testController.debugTest.bind(this)
+      ),
+      vscode.commands.registerCommand(
+        Command.RunTestInTerminal,
+        this.testController.runTestInTerminal.bind(this)
```

Comment:
Let's register the commands in the test controller as well. That way everything is encapsulated in the same class.

Also, this is currently binding a function from the `TestController` class to a `Client` instance, which could potentially lead to weird issues.

Reply:
Oops I didn't know what I was actually doing 😅 Updated it now 👍